deploy/helm/elastic-agent/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: elastic-agent
+description: Elastic-Agent Helm Chart
+kubeVersion: ">= 1.27.0-0"
```

Comment:
What is causing this version constraint? A comment on why that is the minimum version in the file would be very nice to have.

Reply:
tbh nothing in particular, back at the time I was writing the Helm chart this was the older in life k8s version which is now EoL, so happy to change that 🙂

Reply:
Sounds like a good reason. Just didn't know if a k8s feature was required at that level.